pkg/k8s/pod.go

```diff
@@ -84,6 +85,25 @@ func PodIsDaemonSet(pod *v1.Pod) bool {
 			return true
 		}
 	}
+
+	return false
+}
+
+// PodCannotBeDisrupted returns true if the pod cannot be forcibly drained from
+// a node.
+func PodCannotBeDisrupted(pod *v1.Pod) bool {
+	for annotationName, annotationValue := range pod.ObjectMeta.Annotations {
+		if annotationName != doNotDisruptAnnotation {
+			continue
+		}
+
+		if annotationValue != "true" {
```

Comment:
Do we care if the value is `true`? Could make this a little bit foolproof by just checking if the annotation key is present.

Reply:
If we do decide to keep it, I'd like to see the expected value to be a const

Reply:
I would like to keep it otherwise a situation where `cyclops.atlassian.com/do-not-disrupt: "false"` is possible and to me that's not particularly clear. I'd rather keep it strict and define that in the docs, it doesn't make it any harder to use it.